.gitignore

```diff
@@ -6,6 +6,7 @@
 /pkg/
 /spec/reports/
 /tmp/
+/.gems/
```

Comment:
My bundle is configured to install to `.gems` rather than the system set. It might be better as a "global git ignore".